nextcord/abc.py

```diff
@@ -684,6 +685,13 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
             denied = Permissions.all_channel()
             base.value &= ~denied.value
 
+        # if you are timed out then you lose all permissions except view_channel and read_message_history
+        if obj.communication_disabled_until != None:
+            denied = Permissions.none()
+            denied.view_channel = True
+            denied.read_message_history = True
+            base.value &= denied.value
```

Comment:
`denied` doesn't really represent what this does, since those are the only allowed permissions